nepali-calendar-gs-extension@subashghimire.info.np/utils/NepaliDateConverter.js

```
@@ -1,132 +1,186 @@
+  84: {
+    monthDays: [31, 31, 32, 31, 31, 30, 30, 30, 29, 30, 30, 30],
+    totalDays: 365,
+  },
+  85: {
+    monthDays: [31, 32, 31, 32, 30, 31, 30, 30, 29, 30, 30, 30],
+    totalDays: 365,
+  },
+};
+
+export const getYearDataFromCache = (year, extensionPath) => {
+  if (!year || isNaN(year)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)